### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by markulrich**

### CODE REVIEW

File: App.tsx [hunk 1]
```diff
@@ -34,12 +36,15 @@ const Navigator = createStackNavigator(
...
+    MyProfile: {
+      screen: MyProfile
+    },
     VideoPreview: {
       screen: VideoPreview
     }
   } as { [key in RouteName]: any }, // TODO: once react-nav types in, edit
   {
-    initialRouteName: "Home"
+    initialRouteName: RouteName.LogIn
```
Comment (tinaroh) [inline App.tsx]: Revert revert!

File: src/components/pages/LogIn.tsx [hunk 2]
```diff
@@ -46,7 +46,7 @@ class LogInView extends React.Component<LogInProps> {
         <Text>This is the login page.</Text>
         <Button
           title="Cancel"
-          onPress={() => this.props.navigation.navigate("Home")}
+          onPress={() => this.props.navigation.navigate(RouteName.MyProfile)}
```
Comment (tinaroh) [inline src/components/pages/LogIn.tsx]: Same, revert

File: src/components/pages/MyProfile.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
```
Comment (tinaroh) [inline src/components/pages/MyProfile.tsx]: Can we add a comment?

File: src/components/pages/Profile.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
+import * as React from "react";
```
Comment (tinaroh) [inline src/components/pages/Profile.tsx]: Comment

File: src/components/pages/Profile.tsx [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+type OwnProps = {
+  navigation: any;
+  member: Member;
+  isOwnProfile: boolean;
+};
+
+type StateProps = {};
+
+type DispatchProps = {
+  mint: () => void;
```
Comment (tinaroh) [inline src/components/pages/Profile.tsx]: Remove "mint" code from Profile and leave only in MyProfile

**Comment by markulrich** [comment]
screenshot_20180617-172818

File: src/components/pages/Profile.tsx (hunk 5 above)
Comment (markulrich) [reply src/components/pages/Profile.tsx]: Will move both to `actions/`

File: src/store/reducers/members.ts [hunk 6]
```diff
@@ -112,6 +112,12 @@ export class Member {
     );
   }
 
+  public videoUri(): string {
+    return `https://storage.googleapis.com/raha-video/${
```
Comment (rahulgi) [inline src/store/reducers/members.ts]: Can we make this URI configurable depending on prod/test?

**Pull request merged**